packages/graphql_flutter/pubspec.yaml

```diff
@@ -25,4 +25,4 @@ dev_dependencies:
   test: ^1.5.3
 
 environment:
-  sdk: ">=2.2.0 <3.0.0"
+  sdk: ">=2.1.0 <3.0.0"
```

Comment:
Proof that CI is working
> Because graphql_flutter depends on graphql >=1.0.1-beta which requires SDK version >=2.2.0 <3.0.0, version solving failed.